packages/api-plugin-products/README.md

```diff
@@ -8,6 +8,159 @@
 
 Products plugin for the [Reaction API](https://github.com/reactioncommerce/reaction)
 
+### Example on how to use Filter Conditions
+
+We have a query endpoint defined in this plugin which allows us to query products collection based on the input GraphQL conditions object. This query endpoint is defined as `filterSearchProducts.js` and it calls the `generateFilterQuery` function from the `api-utils` plugin to generate the MongoDB filter query.
```

Comment:
These instructions are just what I was looking for

Reply:
Ok, I assume we are good with these updated readme